lib/yt/associations/has_reports.rb

```diff
@@ -225,6 +225,13 @@ def has_report(metric, type)
 
     private
 
+      def deprecated_alias(deprecated, replacement)
+        define_method deprecated do |options = {}|
+          warn "##{deprecated} has been deprecated. please use ##{replacement}"
+          send replacement, options
+        end
+      end
```

Comment:
@kangkyu this is a great idea.

However, let's not deprecate those methods in this PR, since those are methods that we are using in the tests and in the documentation.

If we decide to deprecate them, then we'll have to change those as well.

For now you can just `alias` or `alias_method`